### PR TITLE
docs(structured list) feature flag usage, style, code tab update

### PR DESCRIPTION
**Pull request by laurenmrice**
Closes [#4224](https://github.com/carbon-design-system/carbon-website/issues/4224)

This PR updates the Structured list component Usage, Style, and Code tabs to include feature flag information and images. The content on the Usage and Style tabs have also been updated with the new content templates.

Note: Links in various places to the "Feature Flag tab" have not been updated yet, because that page is not live yet.

---

### Changelog

**New**

Usage:
- Page tag
- Content
- Images

Style:
- Content
- Images

Code:
- Resource card tag
- Content

**Changed**

- Content and images

**Removed**

- Images and content that needed to be taken out

### CODE REVIEW

**Comment by laurenmrice** [comment]
**Need dev help**

Usage tab: 
- [ ] There needs to be a Medium height size, Blue tag, that says `Feature flag`above the description like below: 
🟩 Usage tab tag spacing specs

Code tab:
- [ ] There needs to be a Medium height size, Blue tag, that says `Feature flag` in the React resource card like below:
🟩 Resource card with tag spec

**Review by Kritvi-bhatia17 — COMMENTED** [review]
Hi @laurenmrice, I just realized there’s an issue with displaying images with tabs and without tabs stacked below each other. We might need to explore this a bit.
  
image

**Comment by aagonzales** [comment]
@Kritvi-bhatia17 @laurenmrice I'm also concerned about the tabbed then stacked images we have going on here. Especially in the interactive state section, its a lot of scrolling and the context kind of gets lost in there. Would it be weird to tab the background modifier image as well? Just to help consolidate things? 

image

Another approach I was thinking of was to just move all the "background modifier" color section into on h3 color section
image


These are just thoughts ... we can always come back to this later. We don't need to block this PR while we think on this. This is one of the more tricky Style sequences I've seen.

File: src/pages/components/structured-list/style.mdx
Comment (alisonjoseph) [reply src/pages/components/structured-list/style.mdx:125]: Yea, I think you mean the `<Title>This is a title</Title>` component.
https://gatsby-theme-carbon.vercel.app/components/Title/

Comment (aagonzales) [reply src/pages/components/structured-list/style.mdx:125]: Yes, that's it!

Comment (laurenmrice) [reply src/pages/components/structured-list/style.mdx:125]: Nice!

**Comment by laurenmrice** [comment]
@aagonzales Yeah, this part has been really tricky to deal with and I agree it currently looks strange. I will try one of those suggestions out.

**Review by aagonzales — APPROVED** [review]
It looks good! The only thing I saw was on the style tab with the image under `With background interactive state color` the tabbed image kind of jumps some when tabbing between them. Not really a blocker though so I'm going to go ahead and approve.

**Comment by laurenmrice** [comment]
I see it! I am fixing it before it merges in. Thank you!

**Review by sstrubberg — APPROVED** [review]

**Pull request merged**
